src/run_server.sh

```diff
@@ -187,6 +190,9 @@ function set_variables() {
     MOD_NAMES=${MOD_NAMES:-""}
     MOD_WORKSHOP_IDS=${MOD_WORKSHOP_IDS:-""}
 
+    # Set the Maps to use
+    MAP_NAMES=${MAP_NAMES:-""}
```

Comment:
Considering that `Muldraugh, Ky` is the default according to the [Documentation](https://pzwiki.net/wiki/Server_Settings), maybe we should set it to that here.

```suggestion
    MAP_NAMES=${MAP_NAMES:-"Muldraugh, Ky"}
```